query-engine/core/src/schema_builder/input_types/field_filter_types.rs

```diff
@@ -256,6 +267,28 @@ fn string_filters(mapped_type: InputType) -> impl Iterator<Item = InputField> {
     .into_iter()
 }
 
+fn json_filters(ctx: &mut BuilderContext) -> impl Iterator<Item = InputField> {
+    // TODO: also add json-specific "keys" filters
```

Comment:
Todo still relevant?

Reply:
Yes, there's no `keys` filter yet. I also added a TODO for `json_type` which is not there yet